spec/time.spec.js

```diff
@@ -32,37 +37,692 @@ describe('time', function() {
+            var anim = new anm.Animation();
+            var root = new anm.Element('root');
+            anim.setDuration(10);
+            anim.add(root);
+
+            // anim: 0-----1-----2-----3-----4-----5-----6-
+            // root: 0-----1-·········----2-----3-----4----
+
+            anim.at(1.0, function() { root.pause(); });
+            anim.at(2.0, function() { root.continue(); });
+            anim.tick(1.2); // 1.2
+            expect(root.getTime()).toBe(1.2);
+            anim.tick(1.5); // 2.7 // yet paused, but continue called now
+            expect(root.getTime()).toBe(1.2);
+            anim.tick(1.0); // 3.7
+            // time when root was paused + time passed in animation since last continue call
+            expect(root.getTime()).toBe((3.7 - 2.7) + 1.2);
+        });
+
+        it('time pauses both in the element and its children when `affectsChildren` is `true` (default)', function() {
+            var anim = new anm.Animation();
+            var root = new anm.Element('root');
+            var child = new anm.Element('child');
+            root.add(child);
+            anim.add(root);
+
+            anim.setDuration(10);
+
+            // anim: 0-----1-----2-----3-----4-----5-----6-
+            // root: 0-----1·······························
+            // chld: 0-----1·······························
+
+            anim.at(1.0, function() { root.pause(); });
+            anim.tick(0.5); // 0.5
+            expect(child.getTime()).toBe(0.5);
+            anim.tick(0.5); // 1.0
+            expect(child.getTime()).toBe(1.0);
+            anim.tick(1.0); // 2.0
+            expect(child.getTime()).toBe(1.0);
+            anim.tick(1.0); // 3.0
+            expect(child.getTime()).toBe(1.0);
+        });
+
+        it('time pauses and continues both in the element and its children when `affectsChildren` is `true` (default)', function() {
+            var anim = new anm.Animation();
+            var root = new anm.Element('root');
+            var child = new anm.Element('child');
+            root.add(child);
+            anim.add(root);
+
+            anim.setDuration(10);
+
+            // anim: 0-----1-----2-----3-----4-----5-----6-
+            // root: 0-----1·····1-----2-----3-----4-----5-
+            // chld: 0-----1·····1-----2-----3-----4-----5-
+
+            anim.at(1.0, function() { root.pause(); });
+            anim.at(2.0, function() { root.continue(); });
+            anim.tick(0.5); // 0.5
+            expect(child.getTime()).toBe(0.5);
+            anim.tick(0.5); // 1.0
+            expect(child.getTime()).toBe(1.0);
+            anim.tick(1.0); // 2.0
+            expect(child.getTime()).toBe(1.0);
+            anim.tick(1.0); // 3.0
+            expect(child.getTime()).toBe(2.0);
+        });
+
+        xit('time pauses in the element and its children with bands, when `affectsChildren` flag is `true` (default)', function() {
```

Comment:
`xit` tests are disabled tests, they are yet failing, there's 8 of such for the moment:

![image](https://cloud.githubusercontent.com/assets/110920/13154200/1c771b84-d678-11e5-9e23-05049a6e8c4a.png)